### PR TITLE
Miscellaneous Fixes

### PR DESCRIPTION
**Pull request by curtisf**

### CODE REVIEW

**Comment by Nenkai** [comment]
👌

File: docs/install_windows.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ In text:
 
 If you followed through correctly, you've got Git set up in a way that allows you to perform setup and future maintenance without hitches.
 
-Now, you'd want to figure out where you want the bot to be downloaded. Something simple where you can find it is good, AKA not in the system internals (Possibly some other drive, though **NOT A FLASH DRIVE**) or optionally your Desktop could be fine. When you've found the location of your choice, Shift+Right Click in that particular location and select "Open command window here". Then clone the WildBeast Git repository with `#!bash git clone https://github.com/TheSharks/WildBeast.git && cd WildBeast`.
+Now, you'd want to figure out where you want the bot to be downloaded. Something simple where you can find it is good, AKA not in the system internals (Possibly some other drive, though **NOT A FLASH DRIVE**) or optionally your Desktop could be fine. When you've found the location of your choice, Shift+Right Click in that particular location and select "Open command window here". Then clone the WildBeast Git repository with `#!bash git clone https://github.com/TheSharks/WildBeast && cd WildBeast`.
```
Comment (Dougley) [inline docs/install_windows.md]: `.git` is correct.

**Review by Dougley — CHANGES_REQUESTED** [review]

**Comment by curtisf** [comment]
Yes, I changed that because in the Linux Guide, .git isn't used. I'm trying to enforce a bit of uniformity.

**Comment by Dougley** [comment]
Enforce using `.git` then 😅

**Review by Dougley — APPROVED** [review]

**Review by linuswillner — APPROVED** [review]
Looks all good to me. Always appreciate me some typo fixes 😄

**Pull request merged**